src/App.vue

```diff
@@ -132,7 +196,7 @@ export default {
       // upon successful capturing of order, refresh cart, and clear checkout state, then set order state
       this.commerce.Checkout.capture(checkoutId, order,
         (resp) => {
-          this.refreshCart()
+          this.refreshCart();
           this.checkout = null;
           this.order = resp;
```

Comment:
there'd be one more call to `bandit.logReward({reward: this.order.total})` to send us back what the final amt the user ended up ordering (if they did complete)